sdm/src/main/java/com/sap/cds/sdm/configuration/Registration.java

```diff
@@ -94,8 +92,21 @@ private ListenerProvider createCreationFailedListener(
         new CreationChangeSetListener(contentId, cdsRuntime, outboxedAttachmentService);
   }
 
-  protected EventHandler buildCreateHandler(
-      ModifyAttachmentEventFactory factory, ThreadLocalDataStorage storage) {
-    return new SDMCreateEventHandler(factory, storage);
+  //	protected EventHandler buildCreateHandler(ModifyAttachmentEventFactory factory,
+  // ThreadLocalDataStorage storage,PersistenceService persistenceService) {
+  //		SDMService sdmService = new SDMServiceImpl();
+  //		return new SDMCreateEventHandler(factory, storage, persistenceService, sdmService);
+  //	}
+  //
+  //	protected EventHandler buildUpdateHandler(ModifyAttachmentEventFactory factory,
+  // AttachmentsReader attachmentsReader,
+  //											  ThreadLocalDataStorage storage,PersistenceService persistenceService) {
+  //		SDMService sdmService = new SDMServiceImpl();
+  //		return new SDMUpdateEventHandler(factory,attachmentsReader, storage, persistenceService,
+  // sdmService);
+  //	}
```

Comment:
Please remove these commented lines of code

Reply:
done